### DISCUSSION TITLE
Contributing methodology

### DISCUSSION THREAD

**Discussion by kaklik** [Open-Gamma-Project/Gamma-MCA#31] [Q&A] [answered]
Hello, in the readme is the following formulation:

    Notice: The LICENSE does not apply to the name "Gamma MCA" and especially the Gamma MCA logo. These are Copyright 2021 by me, Phoenix1747.

Based on that I suspect that the forked projects should be renamed and rebranded with a new logo. Ok, but what is the best way to handle contributions pull requests from that modified repositories? 

Additional question, what is exactly the logo? This image? 
![image](https://user-images.githubusercontent.com/1194177/217938588-d97e3fcd-5096-469d-8fcf-42af6a6d86e8.png)

**Comment by NuclearPhoenixx** [comment, marked as answer]
> Based on that I suspect that the forked projects should be renamed and rebranded with a new logo. Ok, but what is the best way to handle contributions pull requests from that modified repositories?

Basically this:

- If you fork to make any changes **and** create PRs for them here (like you do atm), that is totally fine. You do not have to remove any logos or branding for this, that would be ridiculous and impede collaboration.
- If you fork and do not make any changes at all (e.g., to archive the repo), that is totally fine as well. You do not have to remove any logos or branding for archiving.
- If you fork and make (significant) changes without contributing these to the original repository, you must remove all the branding. This would be the same as downloading the code and creating a new repo with it, expanding on top of it and at the same time still branding yourself as _the_ Gamma MCA with the same logo and everything. That would be very confusing to users as well.

> Additional question, what is exactly the logo? This image? ![image](https://user-images.githubusercontent.com/1194177/217938588-d97e3fcd-5096-469d-8fcf-42af6a6d86e8.png)

Yes. It's this file, `logo.svg`: https://github.com/OpenGammaProject/Gamma-MCA/blob/main/assets/logo.svg

This also includes all the other images that incorporate the same logo, like the favicons or the file icons in `assets/`.

It's also the profile picture of @OpenGammaProject here on GitHub, which in itself would not grant anybody the right to use it, but since it is included in this GitHub repo, I have to add this rule. Otherwise due to the GNU GPL v3.0, anybody would be allowed to use in accordance to the license.

I'm going to think of a better phrasing and update the README. Hope this clears it up for you for now.
